.ansible-test/inventory.yml

```diff
@@ -0,0 +1 @@
+localhost ec2_kernel=aki-fc8f11cc ec2_state=running ec2_tag_Cname=testbox ec2_ami_launch_index=0 ec2_private_dns_name=ip-testbox.us-west-2.compute.internal ec2_instance_type=m3.medium ec2_sourceDestCheck=true ec2_id=i-b9999999 ec2_private_ip_address=localhost ec2_security_group_names=Web,default,SSH ec2_vpc_id=vpc-c9999999 ec2_state_code=16 ec2_tag_Name=testbox ec2_architecture=x86_64 ec2_hypervisor=xen ec2_key_name=mgkey ec2_root_device_type=ebs ec2_client_token=141031249511895261 ec2_public_dns_name=ec2-testbo.us-west-2.compute.amazonaws.com ec2_monitoring_state=disabled ec2_root_device_name=/dev/sda ec2_tag_Role=testrole ec2_subnet_id=subnet-b9999999 ec2_security_group_ids=sg-9bc871fe,sg-92d56cf7,sg-14cb7271 ec2_placement=us-west-2b ec2_launch_time=2014-09-10T01:28:20.000Z ec2_ip_address=127.0.0.1 ec2_image_id=ami-b9999999 ec2_region=us-west-2 ec2_tag_Level=dev ec2_virtualization_type=paravirtual ec2_tag_ShardKey=testing-001 ec2_dns_name=ec2-testbox.compute.amazonaws.com
```

Comment:
The `.ansible-test` directory and files appear to be unrelated to the PR and should be removed.